<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TournamentApp</class>
 <widget class="QMainWindow" name="TournamentApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Padel Tournament Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="centralLayout">
    <item>
     <widget class="QStackedWidget" name="stack">
      <property name="currentIndex">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>980</width>
     <height>22</height>
    </rect>
   </property>
    <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="save_action"/>
    <addaction name="load_action"/>
     <addaction name="separator"/>
    <addaction name="csv_action"/>
    <addaction name="pdf_action"/>
     <addaction name="separator"/>
    <addaction name="exit_action"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="theme_action"/>
    <addaction name="auto_action"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
  </widget>
  <action name="save_action">
   <property name="text">
    <string>Save…</string>
   </property>
  </action>
  <action name="load_action">
   <property name="text">
    <string>Load…</string>
   </property>
  </action>
  <action name="csv_action">
   <property name="text">
    <string>Export CSV…</string>
   </property>
  </action>
  <action name="pdf_action">
   <property name="text">
    <string>Export PDF…</string>
   </property>
  </action>
    <action name="exit_action">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="theme_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dark mode</string>
   </property>
  </action>
  <action name="auto_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto-update rankings</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
